tools/e2etesting/ReplaceVariablesInAppSettings.ps1

```diff
@@ -70,7 +70,7 @@ if (!$ImageTag) {
 }
 
 if (!$ImageNamespace) {
-   $ImageNamespace="public"
+   $ImageNamespace="main"
```

Comment:
Will release still publish to public?  Better remove tests for master and main branch from ifs and only let release branches build to public.  Although not sure why we cannot build release builds from main.

Reply:
Release pipeline changes are still in process, idea here is that all the default builds should go to Industrailiotdev on its specified branch name. Release builds can go to public.

Reply:
This is specially needed for staging nightly runs.

Reply:
Discussed this offline with Marc. I plan to publish a spec detailing this workflow.